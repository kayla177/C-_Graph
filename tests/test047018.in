ENTITY KNJW1HG5 name_KNJW journal
ENTITY G9Y0N161 name_G9Y0 institution
ENTITY Z92EQYHF name_Z92E paper
ENTITY 0CYSE59T name_0CYS conference
ENTITY TXKT7RC1 name_TXKT author
ENTITY 9AG6KS10 name_9AG6 journal
ENTITY 8P7UFTLL name_8P7U institution
ENTITY F9XT9937 name_F9XT journal
ENTITY DOR05IBS name_DOR0 institution
ENTITY L3HC3U1V name_L3HC paper
RELATIONSHIP G9Y0N161 published_in L3HC3U1V 5.1
RELATIONSHIP DOR05IBS cited_by KNJW1HG5 7.6
RELATIONSHIP G9Y0N161 cited_by DOR05IBS 7.0
RELATIONSHIP KNJW1HG5 authored_by L3HC3U1V 0.8
RELATIONSHIP 9AG6KS10 cited_by L3HC3U1V 3.4
RELATIONSHIP F9XT9937 authored_by Z92EQYHF 0.9
RELATIONSHIP Z92EQYHF collaborated_with G9Y0N161 2.9
RELATIONSHIP 0CYSE59T published_in KNJW1HG5 5.4
RELATIONSHIP F9XT9937 authored_by G9Y0N161 1.0
RELATIONSHIP TXKT7RC1 collaborated_with 8P7UFTLL 2.2
RELATIONSHIP KNJW1HG5 presented_at TXKT7RC1 7.7
RELATIONSHIP 0CYSE59T authored_by L3HC3U1V 2.3
RELATIONSHIP 0CYSE59T authored_by Z92EQYHF 8.0
RELATIONSHIP F9XT9937 affiliated_with L3HC3U1V 4.7
RELATIONSHIP DOR05IBS presented_at 8P7UFTLL 0.7
PATH Z92EQYHF G9Y0N161
DELETE F9XT9937
DELETE KNJW1HG5
HIGHEST
PRINT L3HC3U1V
DELETE L3HC3U1V
DELETE 0CYSE59T
PATH TXKT7RC1 8P7UFTLL
PRINT 8P7UFTLL
PATH G9Y0N161 8P7UFTLL
EXIT
