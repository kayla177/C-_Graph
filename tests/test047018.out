success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
Z92EQYHF G9Y0N161 2.9
success
success
0CYSE59T TXKT7RC1 20.8
G9Y0N161 9AG6KS10 0CYSE59T
success
success
TXKT7RC1 8P7UFTLL 2.2
TXKT7RC1 DOR05IBS
G9Y0N161 DOR05IBS 8P7UFTLL 7.7
